physionet-django/notification/templates/notification/email/mailto_contact_reference.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}{% autoescape off %}
+Dear Prof. {{ application.reference_name }},
```

Comment:
Not sure the hardcoded "Prof" makes sense...it would be better to ask for the title in the form if we are going to use it, but no problem if you'd like to include for now.

Reply:
He said he always defaults to Prof. since this is a mailto link to generate his own email, he can change this afterwards in his own email client